packages/next-ui/Row/ColumnOne/variant/VariantMessage.tsx

```diff
@@ -0,0 +1,26 @@
+import { Portal } from '@mui/material'
+import { MessageSnackbar, MessageSnackbarProps } from '../../../Snackbar/MessageSnackbar'
+
+export type VariantMessageProps = MessageSnackbarProps
+
+export function VariantMessage(props: VariantMessageProps) {
+  const { sx, ...rest } = props
+
+  return (
+    <Portal>
+      <MessageSnackbar
+        open
+        variant='pill'
+        severity='info'
+        {...rest}
+        sx={[
+          {
+            pointerEvents: 'none',
+            '& > *': { pointerEvents: 'auto' },
+          },
+          ...(Array.isArray(sx) ? sx : [sx]),
+        ]}
+      />
+    </Portal>
```

Comment:
I would assume the Snackbar is already rendered in a Portal? Also the Portal component should be moved to the Impl side if this is really necessary.